pylink/library.py

```diff
@@ -83,13 +83,22 @@ class Library(object):
         'JLINK_SetFlashProgProgressCallback'
     ]
 
-    # Linux/MacOS: The JLink library name without any prefix like lib,
-    # suffix like .so, .dylib or version number.
-    JLINK_SDK_NAME = 'jlinkarm'
+    JLINK_SDK_NAME = 'libjlinkarm'
+    """On Linux and macOS, represents the library file name prefix
+    (the part just before .so or .dylib).
+
+    This is the value appropriate when calling the find_library_{linux,darwin}()
+    functions bellow.
+
+    Note: this "constant" is also used by downstream projects,
+    and should therefore be considered public (frozen) API.
+    """
```

Comment:
```suggestion
    # On Linux and macOS, represents the library file name prefix
    # (the part just before .so or .dylib).
    #
    # This is the value appropriate when calling the find_library_{linux,darwin}()
    # functions below.
    #
    # Note: this "constant" is also used by downstream projects,
    # and should therefore be considered public (frozen) API.
    JLINK_SDK_NAME = 'libjlinkarm'
```
I would change this here to make the style consistent with the rest of the file; comment should also be preceding the definition it comments.

Reply:
It's not a comment, the intent was to attach a docstring to the public *constant* `JLINK_SDK_NAME`.
But the content looks like I'm actually commenting code (and not documenting an API), and the thing eventually smells inconsistent.

So I can:
- either keep the content, but make it a code comment (with `#` and above)
- or fix the content to look like API documentation, and keep the docstring form (with `"` and bellow)

The later may set in stone the value and semantic of `JLINK_SDK_NAME` as a public API. I'm not sure about the best Pythonic practices to achieve this, though.
The former should be enough to avoid breaking downstream again (which upsets us). Let's go with that, the sooner the better with this regression.